stop.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo "Stopping containers..."
+
+source bin/cross_platform_utils.sh
+
+docker compose -f $(getDockerCompose) --env-file ../.env down
```

Comment:
For it to work properly you must change the `.env` location to `.` instead of `../` .